doc/ad_code_values/biden/12/NB.md

```diff
@@ -0,0 +1,10 @@
+---
+value: NB 
+name: New 'Battleground' States
```

Comment:
This will crash the YAML parser unescaped – can you use smart quotes instead like we have had to in `ha` and `sa` for Trump? Alternatively you could escape the single quotes here by doubling them as `''`.

Reply:
Will adjust!